mirrormanager2/templates/fedora/master_noauth.html

```diff
@@ -28,6 +28,12 @@ <h4 class="hide">Main Site Links:</h4>
           <ul>
             <li id="homeTab"><a href="{{url_for('index')}}">Home</a></li>
             <li id="mirrorsTab"><a href="{{url_for('list_mirrors')}}">Mirrors</a></li>
+            {% if config['SHOW_STATISTICS'] %}
+            <li id="statsTab"><a href="{{url_for('statistics')}}">Statistics</a></li>
+            {% endif %}
+            {% if config['SHOW_MAPS'] %}
+            <li id="mapsTab"><a href="{{url_for('maps')}}">Maps</a></li>
+            {% endif %}
```

Comment:
The master template extends the "master_noauth.html" template, so that the links are always visible (if enabled).